packages/json-machete/src/healJSONSchema.ts

```diff
@@ -399,6 +410,9 @@ export async function healJSONSchema(
             }
           }
         }
+        if (subSchema.type === 'array' && subSchema.anyOf) {
```

Comment:
shouldn't we make it a little more verbose and fail if the original schema includes a logical fail